staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -828,8 +838,12 @@ func (meal *cfgMeal) finishQueueSetReconfigsLocked() {
 		if limited := plState.pl.Spec.Limited; limited != nil {
 			if qCfg := limited.LimitResponse.Queuing; qCfg != nil {
 				meal.maxWaitingRequests += int(qCfg.Queues * qCfg.QueueLengthLimit)
+
+				plState.maxSeats = uint64(math.Max(float64(1), math.Min(math.Ceil(float64(concurrencyLimit)*priorityLevelMaxSeatsPercent),
+					float64(int32(concurrencyLimit)/plState.pl.Spec.Limited.LimitResponse.Queuing.HandSize))))
```

Comment:
Can `HandSize` be 0?

Reply:
This is guarded by checking  `limited.LimitResponse.Queuing` above, so I don't think so?

Reply:
(also there's validation for handsize <= 0)